tests/src/io/openliberty/tools/eclipse/test/it/LibertyPluginSWTBotGradleTest.java

```diff
@@ -194,6 +198,9 @@ public void testDashboardStartAction() {
         // Validate application stopped.
         validateApplicationOutcome(GRADLE_APP_NAME, false, testAppPath + "/build");
 
+        // unset the preferences
```

Comment:
Why do we need to do so many unsets?   Given that the wrapper should take precedence over the preference it seems the only reason we'd need to unset would be to run a test where we intended to have the preference unset.  I'd think that might just be one test which could do the unset itself?